package.json

```diff
@@ -46,13 +48,16 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
     "eslint-plugin-react": "^7.29.4",
+    "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.5.0",
     "json-server": "^0.17.0",
     "node-fetch": "^3.2.3",
     "nodemon": "^2.0.15",
     "style-loader": "^3.3.1",
+    "svg-url-loader": "^7.1.1",
```

Comment:
Can remove url-loader and svg-url-loader dependencies if they are no longer used?